deploy/helm/sumologic/templates/metrics/remote-write-proxy/deployment.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if .Values.sumologic.metrics.remoteWriteProxy.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "sumologic.metadata.name.remoteWriteProxy.deployment" . }}
+  labels:
+    app: {{ template "sumologic.labels.app.remoteWriteProxy.deployment" . }}
+    {{- include "sumologic.labels.common" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "sumologic.labels.app.remoteWriteProxy.pod" . }}
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: {{ template "sumologic.labels.app.remoteWriteProxy.pod" . }}
+    spec:
+      containers:
+      - name: nginx
+        image: nginx
+        ports:
+        - containerPort: 80
+        resources:
+          limits:
+            cpu: 1000m
+            memory: 256Mi
+          requests:
+            cpu: 100m
+            memory: 32Mi
+        livenessProbe:
+          tcpSocket:
+            port: 80
+          initialDelaySeconds: 30
+          periodSeconds: 10
+          timeoutSeconds: 5
+          successThreshold: 1
+          failureThreshold: 6
+        readinessProbe:
+          tcpSocket:
+            port: 80
+          initialDelaySeconds: 5
+          periodSeconds: 5
+          timeoutSeconds: 3
+          successThreshold: 1
+          failureThreshold: 3
```

Comment:
Pretty custom thresholds :)

Isn't the 3s readiness timeout a bit too aggressive?

Reply:
Shamelessly copied from [Bitnami's nginx Helm chart.](https://github.com/bitnami/charts/blob/f392ed0715f31f5ed78eae15a5b4da490e9ebb8e/bitnami/nginx/values.yaml#L215).

I don't think 3s is very aggressive for nginx specifically, it needs to be very overloaded to miss that.